### PR TITLE
Added the use case for predictions

### PR DESCRIPTION
**Pull request by KuzeyOzturac**

### CODE REVIEW

**Review by Swithord — CHANGES_REQUESTED** [review]
missing input data class

File: src/use_case/PredictionsInteractor.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class PredictionsInteractor implements PredictionsInputBoundary{
+    final PredictionsDataAccessInterface predictionsDataAccessInterface;
+    final PredictionsOutputBoundary predictionsPresenter;
+
+    public PredictionsInteractor(PredictionsDataAccessInterface predictionsDataAccessInterface, PredictionsOutputBoundary predictionsPresenter){
+        this.predictionsDataAccessInterface = predictionsDataAccessInterface;
```
Comment (nsaroiu) [inline src/use_case/PredictionsInteractor.java:14]: test poopoo peepee

File: src/use_case/PredictionsOutputData.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package use_case;
+
+import entity.Location;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class PredictionsOutputData {
+    //direction string,
```
Comment (nsaroiu) [inline src/use_case/PredictionsOutputData.java:9]: test 2

**Review by nsaroiu — CHANGES_REQUESTED** [review]
hey, make sure you address the changes

File: src/use_case/PredictionsOutputData.java (hunk 2 above)
Comment (KuzeyOzturac) [reply src/use_case/PredictionsOutputData.java:10]: Change addressed and committed!

**Review by nsaroiu — COMMENTED** [review]
Thank you for addressing the changes

**Pull request closed without merging**